config.go

```diff
@@ -117,6 +117,16 @@ type RuntimeConfig interface {
 	//	foo := wazero.NewRuntimeWithConfig(context.Background(), config)
 	// 	bar := wazero.NewRuntimeWithConfig(context.Background(), config)
 	WithCompilationCache(CompilationCache) RuntimeConfig
+
+	// WithCustomSections toggles parsing of "custom sections". Defaults to false.
+	//
+	// When enabled, it is possible to retrieve custom sections from a CompiledModule:
+	//
+	//	config := wazero.NewRuntimeConfig().WithCustomSections(true)
+	//	r := wazero.NewRuntimeWithConfig(context.Background(), config)
+	//  c, err := r.CompileModule(ctx, wasm)
+	//  customSections := c.CustomSections()
```

Comment:
wrong indentation? 😉

Reply:
I think I followed the style of the doc strings above it? the example is indented 1 tab